pkg/disttask/importinto/scheduler.go

```diff
@@ -136,6 +139,8 @@ type ImportSchedulerExt struct {
 	// It may be changed when we switch to a new task or switch to a new owner.
 	currTaskID            atomic.Int64
 	disableTiKVImportMode atomic.Bool
+
+	store kv.Storage
```

Comment:
```suggestion
	storeWithPD tidbkv.StorageWithPD
```
we don't need depends on a heavy `store`.

at task executor part we only need a tidbkv.StorageWithPD and a `keyspaceGetter`, but we can change it later.